_posts/2020-03-26-amazon-connect.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Amazon Connect: A work from home call center"
+author: blongden
+---
+
+![Photo by Andrea Piacquadio from Pexels](/images/amazon-connect/woman-using-white-headphones-using-macbook-3885801.jpg)
+
+One week in to this brave new world of working from home and we are settling in quite nicely to the new 'normal', here at Infinity Works. We're lucky that much of the day to day tooling we use as consultants is very remote friendly, so aside from a few initial issues and a slightly feeling of cabin fever, we have all managed to adjust relatively easily. All our customers are going through the same thing. One of the big issues that we have seen over the past week is the challenge in getting call centre staff to be able to perform their duties out of the office.
+
+As cloud natives, we're experts at working in AWS to develop solutions that take advantage of the full power of the cloud and of the services that AWS has to offer. We have embraced server-less technology and use it to great effect with our partners to achieve amazing things, all largely without the need for the maintenance of traditional servers. Given the presence of customer data in the cloud it would seem logical that there would be a service to allow contact to those customers via means other than the obvious 'cloud' interfaces through web and mobile.
```

Comment:
Server-less or serverless? Not sure of the 'official etiquette' here. We use 'serverless' for the meetup.